TestingApp/Main.storyboard

```diff
@@ -22,6 +22,7 @@
                                 <subviews>
```

Comment:
It seems like the changes on this file are part of the other PR about adding accessibility IDs for AT

Reply:
yes, this is changes from at/0, but I needed this in this PR to have proper build for acceptance tests.